### PR TITLE
Backports 0.15 pr1

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: qa/pull-tester/rpc-tests.py [hunk 1]
```diff
@@ -39,8 +39,8 @@
 #If imported values are not defined then set to zero (or disabled)
 if 'ENABLE_WALLET' not in vars():
     ENABLE_WALLET=0
-if 'ENABLE_BITCOIND' not in vars():
-    ENABLE_BITCOIND=0
+if 'ENABLE_DASHD' not in vars():
```
Comment (UdjinM6) [inline qa/pull-tester/rpc-tests.py]: Should not dashify env vars (or should dashify them everywhere but in another PR).

File: qa/pull-tester/rpc-tests.py [hunk 2]
```diff
@@ -295,8 +295,8 @@ def run_tests(test_list, src_dir, build_dir, exeext, jobs=1, enable_coverage=Fal
         BOLD = ('\033[0m', '\033[1m')
 
     #Set env vars
-    if "BITCOIND" not in os.environ:
-        os.environ["BITCOIND"] = build_dir + '/src/bitcoind' + exeext
+    if "DASHD" not in os.environ:
```
Comment (UdjinM6) [inline qa/pull-tester/rpc-tests.py]: same for this env var too

File: qa/pull-tester/tests_config.ini.in [hunk 3]
```diff
@@ -13,6 +13,6 @@ EXEEXT=@EXEEXT@
 [components]
 # Which components are enabled. These are commented out by `configure` if they were disabled when running config.
 @ENABLE_WALLET_TRUE@ENABLE_WALLET=true
-@BUILD_BITCOIN_UTILS_TRUE@ENABLE_UTILS=true
-@BUILD_BITCOIND_TRUE@ENABLE_BITCOIND=true
+@BUILD_DASH_UTILS_TRUE@ENABLE_UTILS=true
+@BUILD_DASHD_TRUE@ENABLE_DASHD=true
```
Comment (UdjinM6) [inline qa/pull-tester/tests_config.ini.in]: same for both lines

File: qa/pull-tester/rpc-tests.py [hunk 4]
```diff
@@ -173,10 +92,15 @@
     'p2p-compactblocks.py',
     'sporks.py',
 ]
-if ENABLE_ZMQ:
-    testScripts.append('zmq_test.py')
 
-testScriptsExt = [
+ZMQ_SCRIPTS = [
+    # ZMQ test can only be run if bitcoin was built with zmq-enabled.
```
Comment (UdjinM6) [inline qa/pull-tester/rpc-tests.py]:
```suggestion
    # ZMQ test can only be run if Dash Core was built with zmq-enabled.
```

File: qa/pull-tester/rpc-tests.py [hunk 5]
```diff
@@ -204,44 +128,126 @@
...
+def run_tests(test_list, src_dir, build_dir, exeext, jobs=1, enable_coverage=False, args=[]):
+    BOLD = ("","")
+    if os.name == 'posix':
+        # primitive formatting on supported
+        # terminal via ANSI escape sequences:
+        BOLD = ('\033[0m', '\033[1m')
+
+    #Set env vars
+    if "BITCOIND" not in os.environ:
+        os.environ["BITCOIND"] = build_dir + '/src/bitcoind' + exeext
```
Comment (UdjinM6) [inline qa/pull-tester/rpc-tests.py]:
```suggestion
        os.environ["BITCOIND"] = build_dir + '/src/dashd' + exeext
```

File: src/qt/intro.cpp [hunk 6]
```diff
@@ -125,16 +125,34 @@ Intro::Intro(QWidget *parent) :
     ui->setupUi(this);
     ui->welcomeLabel->setText(ui->welcomeLabel->text().arg(tr(PACKAGE_NAME)));
     ui->storageLabel->setText(ui->storageLabel->text().arg(tr(PACKAGE_NAME)));
+
+    ui->lblExplanation1->setText(ui->lblExplanation1->text()
+        .arg(tr(PACKAGE_NAME))
+        .arg(BLOCK_CHAIN_SIZE)
+        .arg(2009)
```
Comment (UdjinM6) [inline src/qt/intro.cpp]:
```suggestion
        .arg(2014)
```

File: src/qt/intro.cpp [hunk 7]
```diff
@@ -125,16 +125,34 @@ Intro::Intro(QWidget *parent) :
...
+        .arg(tr("Bitcoin"))
```
Comment (UdjinM6) [inline src/qt/intro.cpp]:
```suggestion
        .arg(tr("Dash"))
```

**Comment by UdjinM6** [comment]
Looks like test failures are semi-related and should be fixed by #2603. Well, at least tests fail in their usual manner https://travis-ci.org/UdjinM6/dash/builds/474659810 after applying the patch.

**Comment by codablock** [comment]
I'd prefer to have https://github.com/dashpay/dash/pull/2603 merged first and this one rebased afterwards.

**Comment by UdjinM6** [comment]
>Otherwise we'd merge this one without tests being run.

You can see test results for this branch with the patch from #2603 applied on top in the link I provided above (https://travis-ci.org/UdjinM6/dash/builds/474659810).

**Pull request merged**